client/src/Month.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+import Week from './Week.jsx';
+
+class Month extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div id = 'month1'>
+        {this.props.month}
+        <table>
+          <tbody>
+            <tr>
+              <td>Su</td>
+              <td>Mo</td>
+              <td>Tu</td>
+              <td>We</td>
+              <td>Th</td>
+              <td>Fr</td>
+              <td>Sa</td>
+            </tr>
+            {/* <Week thisWeek = {['blank', 'blank', 'blank', 'blank', 'blank', 'blank', 'blank']} /> */}
+            {this.props.weeks.map((arr) => {
+              return (<Week selectedCheckoutOnlyDate = {this.props.selectedCheckoutOnlyDate} dates = {this.props.dates} checkInDate = {this.props.checkInDate} checkOutDate = {this.props.checkOutDate} hoveredDate = {this.props.hoveredDate} changedHoveredDate = {this.props.changedHoveredDate} thisWeek = {arr} dateClicked = {this.props.dateClicked}/>);
```

Comment:
This looks great!
One interesting observation is regarding the column length(width) for this line of code, especially for readability purposes to fit within the screen without the need of horizontal scrolling. Not sure if this is a linter or IDE setting.